doc/api/v1/apipie_definitions.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    module ApipieDefinitions
+      extend Apipie::DSL::Concern
+
+      def_param_group :internal_error do
+        error code: 500, desc: 'Internal server error'
+      end
+
+      def_param_group :general_errors do
+        error code: 404, desc: 'Resource not found'
+        param_group :internal_error, Api::V1::ApipieDefinitions
+      end
+
+      def_param_group :validation_errors do
+        param_group :general_errors, Api::V1::ApipieDefinitions
+        error code: 422, desc: 'Validation failed'
+        error code: 400, desc: 'Invalid request'
+      end
```

Comment:
I disagree with the naming of the group `:validation_errors` when in fact it includes 404 and 500 errors. These errors have nothing to do with validations. 
Also, keep things consistent in the description of the errors with the reference documentation of HTTP status codes https://developer.mozilla.org/en-US/docs/Web/HTTP/Status